libtesting-ui/src/main/java/com/mapbox/navigation/testing/ui/http/MockWebServerRule.kt

```diff
@@ -81,10 +85,24 @@ class MockWebServerRule : TestWatcher() {
             block()
         } finally {
             withContext(Dispatchers.IO) {
-                webServer = MockWebServer()
+                retryStarting(previousPort)
                 initDispatcher()
-                webServer.start(previousPort)
             }
         }
     }
+
+    private suspend fun retryStarting(port: Int) {
```

Comment:
Not connected to this PR, right? Just additional stabilizing?

Reply:
I had a problem with a new test where `withoutInternet` block was too short. For some reason port didn't become free after mock http server shotdown, so I had to introduce retry